packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/eslint.config.mjs.template

```diff
@@ -0,0 +1,77 @@
+import o3rConfig from '@o3r/eslint-config';
+import globals from 'globals';
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+// eslint-disable-next-line no-underscore-dangle
+const __filename = fileURLToPath(import.meta.url);
+// __dirname is not defined in ES module scope
+// eslint-disable-next-line no-underscore-dangle
+const __dirname = dirname(__filename);
+
+export default [
+  ...o3rConfig,
+  {
+    name: '<%= packageName %>/report-unused-disable-directives',
+    linterOptions: {
+      reportUnusedDisableDirectives: 'error'
+    }
+  },
+  {
+    name: '<%= packageName %>/projects',
+    languageOptions: {
+      sourceType: 'commonjs',
+      parserOptions: {
+        tsconfigRootDir: __dirname,
+        project: [
+          'testing/tsconfig.spec.json',
+          'tsconfigs/tsconfig.jest.json',
+          'tsconfigs/tsconfig.source.json'
+        ]
+      },
+      ecmaVersion: 12
+    }
+  },
+  {
+    name: '@o3r/globals',
```

Comment:
Not sure you want this rule with this name in a fresh generated SDK.
Maybe you can name it `'<%= packageName %>/globals'`